amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -1906,7 +1906,7 @@ type DomainMetadataChangelog {
 """
 Represents a Colony Funds Claim for all ERC20 tokens (except native chain tokens)
 """
-type ColonyFundsClaim @model @searchable {
```

Comment:
Yeah, as Jakub commented above, it's not being used at the moment and adding it will increase AWS costs. When it needs to be used (probably when the full page is properly redesigned next quarter as sounds likely) then it can be easily added back.

Reply:
"easily" meaning codewise. Devops wise we'll need to re-index all existing entries in the table, otherwise only new ones will be counted.